### PR TITLE
docs: fix error link path in storage e2e test doc

### PR DESCRIPTION
**Pull request by gxglls**
#### What type of PR is this?

/kind documentation

### CODE REVIEW

File: test/e2e/storage/external/README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ filename can be absolute or relative to `--repo-root`. The content of
 the file is used to populate a struct that defines how to test the
 driver. For a full definition of the content see:
 - `struct driverDefinition` in [external.go](./external.go)
-- `struct TestDriver` and the `Cap` capability constants in [testdriver.go](../testsuites/testdriver.go)
+- `struct TestDriver` and the `Cap` capability constants in [testdriver.go](../framework/testsuite.go)
```
Comment (mattcary) [inline test/e2e/storage/external/README.md:10]: It looks like the struct and constants are in ../framework/testdriver.go?

Comment (gxglls) [reply test/e2e/storage/external/README.md]: fixed.

**Comment by mattcary** [comment]
/lgtm

Thanks! I assume we'll have to wait until the code thaw until this can be submitted.

**Pull request merged**